.gitattributes

```diff
@@ -0,0 +1,2 @@
+* text eol=lf
+**/tests/fixtures/*_CRLF/** text eol=crlf
```

Comment:
Would it be better to have a separate `.gitattributes` file right there in the test?

Having it local like that seems like it'd help in terms of moving the code around, etc.

Reply:
I didn't know we can have multiple `.gitattributes`, good idea!